src/components/ArtifactDetailedInfo.tsx

```diff
@@ -383,6 +399,108 @@ const LinkedAdvisories: React.FC<LinkedAdvisoriesProps> = (props) => {
     );
 };
 
+const ErrataAutomationBugCiStatusHumanReadable: {
+    [key in ErrataAutomationBugCiStatus]: string;
+} = {
+    BUG_READY: 'ready',
+    BUG_IN_ADVISORY: 'in advisory',
+    BUG_VERIFIED_TESTED_MISSING: 'Verified:Tested missing',
+};
+
+interface ErrataAutomationProps {
+    artifact: Artifact;
+}
+const ErrataAutomation: React.FC<ErrataAutomationProps> = (props) => {
+    const { artifact } = props;
+    const latestState = artifact.states_eta?.[0];
+    if (_.isNil(latestState)) {
+        return (
+            <Flex className="pf-u-p-lg">
+                <Alert
+                    isInline
+                    isPlain
+                    title="No Errata Tool Automation activity known for this artifact"
+                    variant="info"
+                />
+            </Flex>
+        );
+    }
+    const advs: JSX.Element[] = [];
+    const brokerMessage = latestState.broker_msg_body;
+    const bugs = brokerMessage.bugs;
+    for (const bug of bugs) {
+        const bugCiState =
+            ErrataAutomationBugCiStatusHumanReadable[bug.ci_status] ||
+            'Unknown';
+        advs.push(
+            <Tr key={bug.id}>
+                <Td dataLabel="bug">
+                    <a
+                        href={`${config.et.url}/advisory/${bug.id}`}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        {bug.id}
+                    </a>
+                </Td>
+                <Td dataLabel="state">{bugCiState}</Td>
+                <Td dataLabel="fixed in">{bug.fixed_in_version}</Td>
+                <Td dataLabel="summary">{bug.summary}</Td>
+            </Tr>,
+        );
+    }
+    const brokerMsgUrl: string = new URL(
+        `id?id=${latestState.kai_state.msg_id}&is_raw=true&size=extra-large`,
+        mappingDatagrepperUrl[artifact.type],
+    ).toString();
```

Comment:
Can you please factor this out to a separate function? It's already duplicated in [`ArtifactKaiState.tsx`](https://github.com/fedora-ci/ciboard/blob/e9c5371afa3733810ad80eecf3c3716401d976a0/src/components/ArtifactKaiState.tsx#L288-L291).